### PR TITLE
III-3528 Capture user info and runtime info

### PR DESCRIPTION
**Pull request by LucWollants**
### Added

- Capture user information (ID, sub, uid, uitidv1id)
- Capture runtime information (api_key, api_name)

---
Ticket: https://jira.uitdatabank.be/browse/III-3528

### CODE REVIEW

**Review by jdrieghe — APPROVED** [review]
Looks good to me

File: app/SentryServiceProvider.php [hunk 1]
```diff
@@ -26,20 +25,19 @@ function (Application $app) {
...
-
-                return static function (Throwable $throwable) use ($sentryHub) {
-                    $sentryHub->captureException($throwable);
-                    throw $throwable;
-                };
+                return new SentryErrorHandler(
+                    $app[HubInterface::class],
+                    $app['jwt'] ?? null,
+                    $app['auth.api_key'] ?? null,
+                    $app['api_name'] ?? null
```
Comment (bertramakers) [inline app/SentryServiceProvider.php:34]: This parameter is not nullable according to https://github.com/cultuurnet/udb3-silex/pull/507/files#diff-64fd9397ae1ed3a3e08bd0a9da5c7f02R27 so the default should be an empty string instead. Or make it nullable.

Comment (bertramakers) [reply app/SentryServiceProvider.php:34]: But good idea to also include it! 👍

Comment (LucWollants) [reply app/SentryServiceProvider.php:34]: I made it consistent with other parameters and it's also nullable now, see https://github.com/cultuurnet/udb3-silex/pull/508

**Pull request merged**